common/eth2_network_config/src/lib.rs

```diff
@@ -324,4 +324,4 @@ mod tests {
 
         assert_eq!(testnet, decoded, "should decode as encoded");
     }
-}
+}
```

Comment:
Looks like there's some formatting issues (I suspect it's due to there not being a newline here)
Would you mind getting `cargo fmt` passing so that the rest of CI can run?